src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Vault/SetAzureRmRecoveryServicesAsrVaultContext.cs

```diff
@@ -76,7 +76,7 @@ public override void ExecuteSiteRecoveryCmdlet()
                 catch (CloudException ex)
                 {
                     Logger.Instance.WriteWarning(ex.Message);
-                    throw new Exception(Resources.TryDownloadingVaultFile);
+                    // throw new Exception(Resources.TryDownloadingVaultFile);
```

Comment:
Commented code. Please remove if intended to no longer throw. Also, why was this exception commented out?

Reply:
// code interanally handled the cloud exception thrown earlier.But still there are changes of other exception.
                    // suggesting alternate way to user to unblock if this command is failing.